kayak_core/src/event_dispatcher.rs

```diff
@@ -303,9 +304,27 @@ impl EventDispatcher {
                 InputEvent::CharEvent { c } => event_stream.push(
                     Event::new(current_focus, EventType::CharInput { c: *c })
                 ),
-                InputEvent::Keyboard { key } => event_stream.push(
-                    Event::new(current_focus, EventType::KeyboardInput { key: *key })
-                ),
+                InputEvent::Keyboard { key, is_pressed } => {
+                    // === Modifers === //
+                    match key {
+                        KeyCode::LControl | KeyCode::RControl => self.keyboard_modifiers.is_ctrl_pressed = *is_pressed,
+                        KeyCode::LShift | KeyCode::RShift => self.keyboard_modifiers.is_shift_pressed = *is_pressed,
+                        KeyCode::LAlt | KeyCode::RAlt => self.keyboard_modifiers.is_alt_pressed = *is_pressed,
+                        KeyCode::LWin | KeyCode::RWin => self.keyboard_modifiers.is_meta_pressed = *is_pressed,
```

Comment:
Do you know what macos uses for command? LWin and RWin are the same as command for macos?

Reply:
Yeah it's the same for both.

I tried to suggest renaming it in an [issue](https://github.com/bevyengine/bevy/issues/3240) but some people didn't like the term "meta".

I think it's fine here, though, since we're trying to be sorta like React and the web uses that [terminology](https://www.w3.org/TR/uievents/#dom-keyboardevent-metakey). But let me know if you prefer a different term!

Reply:
Actually if we use "meta" or some other term should we also update our KeyCode enum to match? Or do we remain consistent with Bevy? 🤔

Reply:
I think if we keep LWin and RWin it's fine. I don't want to diverge too much from bevy.

Reply:
Do you want to also change `is_meta_pressed` to `is_win_pressed`? Or are we just keeping the keycodes themselves consistent with bevy/winit?